crates/cargo-lambda-remote/src/lib.rs

```diff
@@ -221,6 +221,9 @@ mod tests {
     async fn default_profile() {
         setup();
 
+        // make sure the default region from environment variable does not override the default from the profile
+        std::env::remove_var("AWS_DEFAULT_REGION");
```

Comment:
<img width="921" alt="Capture d’écran 2024-04-26 à 16 57 48" src="https://github.com/cargo-lambda/cargo-lambda/assets/2571084/84b713bb-d921-4c3e-83f4-dbbc51d9e627">

(because I have a `AWS_DEFAULT_REGION=eu-west-3` env var always defined in my shell... but feel free to remove this if you think it's a bad idea)

Reply:
Done ✅